2021-06-23 08:20:34,057 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-06-23 08:20:34,061 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-06-23 08:21:46,856 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-06-23 08:21:46,858 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-06-23 08:21:52,847 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-06-23 08:21:52,851 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-06-23 14:01:49,009 - gravity_core.wlogger - ERROR - Выход не зафиксирован фотоэлементом
